src/main/java/org/folio/rest/support/InstanceUtil.java

```diff
@@ -0,0 +1,265 @@
+package org.folio.rest.support;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import java.util.ArrayList;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import org.folio.rest.jaxrs.model.AlternativeTitle;
+import org.folio.rest.jaxrs.model.Classification;
+import org.folio.rest.jaxrs.model.Contributor;
+import org.folio.rest.jaxrs.model.ElectronicAccess;
+import org.folio.rest.jaxrs.model.Identifier;
+import org.folio.rest.jaxrs.model.Instance;
+import org.folio.rest.jaxrs.model.Instance.SourceRecordFormat;
+import org.folio.rest.jaxrs.model.InstanceNote;
+import org.folio.rest.jaxrs.model.Publication;
+import org.folio.rest.jaxrs.model.PublicationPeriod;
+import org.folio.rest.jaxrs.model.Series;
+import org.folio.rest.jaxrs.model.Subject;
+import org.folio.rest.jaxrs.model.Tags;
+
+public final class InstanceUtil {
```

Comment:
@RuslanLavrov Let's add documentation for this class for a better understanding of the main purpose of this entity.

Reply:
added, however, objectively the class semantic is obvious, and methods here had already been commented